recipes/fcgi2/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+
+
+nmake /f Makefile.nt
+if errorlevel 1 exit 1
+
+copy cgi-fcgi\Release\cgi-fcgi.exe %PREFIX%\bin
```

Comment:
```suggestion
copy cgi-fcgi\Release\cgi-fcgi.exe %PREFIX%\bin
if errorlevel 1 exit 1
```

Reply:
Updated as suggested in 780fc05cdec3954e4e1db11e4b684619a39e3058